src/Adapter/Module/Repository/ModuleRepository.php

```diff
@@ -40,6 +40,13 @@
  */
 class ModuleRepository extends AbstractObjectModelRepository
 {
+    /** @var string[] */
+    public const ADDITIONAL_ALLOWED_MODULES = [
+        'autoupgrade',
+    ];
+
+    private const COMPOSER_PACKAGE_TYPE = 'prestashop-module';
```

Comment:
```suggestion
    private const COMPOSER_PACKAGE_TYPE = 'prestashop-native-module';
```

I would suggest changing it to a more precise package type. I know about some PrestaShop modules that have the same package type. To avoid problems with third-party modules that use composer, we should be more precise.

I know it requires changes in all the native modules, but this package type doesn't necessarily mean **native** modules.

Reply:
I would like to see other maintainers' opinions about it. Maybe I'm overthinking, but...

https://github.com/friends-of-presta/masterclass/blob/master/composer.json
https://github.com/friends-of-presta/fop_console/blob/dev/composer.json
https://github.com/PrestaShop/example-modules/blob/master/demoextendsymfonyform3/composer.json
https://github.com/PrestaShop/example-modules/blob/master/demoextendsymfonyform1/composer.json
https://github.com/PrestaShop/example-modules/blob/master/demovieworderhooks/composer.json



cc @PrestaShop/prestashop-maintainers

Reply:
@kpodemski As long as those modules are not included in the PrestaShop's composer.json, I guess we're pretty safe

Reply:
@atomiix you're right, we scan composer.lock, not the `/modules/` folder